shell/platform/android/test/io/flutter/embedding/engine/systemchannels/PlatformChannelTest.java

```diff
@@ -0,0 +1,126 @@
+package io.flutter.embedding.engine.systemchannels;
+
+import android.graphics.Rect;
+
+import java.util.ArrayList;
+
+import io.flutter.embedding.engine.dart.DartExecutor;
+import io.flutter.embedding.engine.systemchannels.PlatformChannel;
+import io.flutter.embedding.engine.systemchannels.PlatformChannel.PlatformMessageHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import io.flutter.plugin.common.MethodCall;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@Config(manifest=Config.NONE)
+@RunWith(RobolectricTestRunner.class)
+public class PlatformChannelTest {
+    @Test
+    public void setSystemExclusionRectsSendsSuccessMessageToFramework() throws JSONException {
+        DartExecutor dartExecutor = mock(DartExecutor.class);
```

Comment:
Please denote with comments in each test the part that is setup vs execution vs verification so that developers can more easily check the part they're interested in.